composeApp/src/commonMain/kotlin/com/theseuntaylor/snippy/ui/destinations/home/HomeScreen.kt

```diff
@@ -0,0 +1,133 @@
+package com.theseuntaylor.snippy.ui.destinations.home
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.theseuntaylor.snippy.ui.components.SnippyButton
+import com.theseuntaylor.snippy.ui.components.SnippyVerticalSpacer
+import org.jetbrains.compose.resources.painterResource
+import org.jetbrains.compose.resources.stringResource
+import snippy.composeapp.generated.resources.Res
+import snippy.composeapp.generated.resources.app_name
+import snippy.composeapp.generated.resources.compose_multiplatform
+import snippy.composeapp.generated.resources.create_account
+import snippy.composeapp.generated.resources.features_only_available_with_an_account
+import snippy.composeapp.generated.resources.login
+import snippy.composeapp.generated.resources.privacy_policy
+import snippy.composeapp.generated.resources.shorten
+import snippy.composeapp.generated.resources.shorten_your_links
+import snippy.composeapp.generated.resources.simplify_long_urls
+import snippy.composeapp.generated.resources.url_hint
+
+@Composable
+internal fun HomeScreen(
+    modifier: Modifier = Modifier,
+) {
+    var urlLink by remember { mutableStateOf("") }
```

Comment:
This is analogous to naming something `vehicleCar` or `fruitApple`. `url` or simply `link` is fine.